mmdet3d/datasets/transforms/loading.py

```diff
@@ -718,6 +722,8 @@ class LoadAnnotations3D(LoadAnnotations):
             Defaults to False.
         with_seg_3d (bool): Whether to load 3D semantic masks for points.
             Defaults to False.
+        with_panoptic_seg_3d (bool): Whether to load 3D panoptic masks
```

Comment:
with_panoptic_seg_3d -> with_panoptic_3d, for consistency with with_mask_3d and with_seg_3d